resources/views/messages/welcome.blade.php

```diff
@@ -1 +1 @@
-<?php echo $email->message->body ?>
+<?php echo $body; ?>
```

Comment:
This is not a big deal, but is there any reason this isn't using Blade tags? `{!! $body !!}`